src/app/kvm/views/KVMList/KVMList.tsx

```diff
@@ -39,7 +39,7 @@ const KVMList = (): JSX.Element => {
   const hasVirsh = virshKvms.length > 0;
   const showingLXD = location.pathname.endsWith(urls.kvm.lxd.index);
   const showingVirsh = location.pathname.endsWith(urls.kvm.virsh.index);
-  useWindowTitle("KVM");
+  useWindowTitle(showingLXD ? "LXD" : "Virsh");
```

Comment:
The way this condition works means KVM title will never be rendered as Virsh is always going to be the fallback value.

```suggestion
  const title = showingLXD ? "LXD" : showingVirsh ? "Virsh" : "KVM";
  useWindowTitle(showingLXD ? "LXD" : "Virsh");
```